changelog.d/13308.bugfix

```diff
@@ -0,0 +1 @@
+Fix async get event cache invalidation logic. Contributed by Nick @ Beeper (@fizzadar).
```

Comment:
since #13242 hasn't yet made it into a release, we should just merge their changelogs:

```suggestion
 Use an asynchronous cache wrapper for the get event cache. Contributed by Nick @ Beeper (@fizzadar).
```